pyshtools/shclasses/shcoeffsgrid.py

```diff
@@ -717,6 +806,40 @@ def to_file(self, filename, format='shtools', header=None, **kwargs):
             raise NotImplementedError(
                 'format={:s} not implemented.'.format(repr(format)))
 
+    def to_netcdf(self, filename, title='', description=''):
+        """Return the coefficient data as a netcdf formatted file or object.
+        
+        Usage
+        -----
+        x.to_netcdf([filename, title, description])
+        
+        Parameters
+        ----------
+        filename : str
+            Name of the output file.
+        title : str, optional, default = ''
+            Title of the dataset
+        description : str, optional, default = ''
+            Description of the data.
+        """
+        
+        if not filename[-3:] == '.nc':
```

Comment:
Same comment as above: I think that if the user provides a filename with an extension that is not `.nc` that we shouldn't append the extension `.nc`.

Reply:
Deleted as well.